# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchStreamUtils.h
# Commit: 66b279e5a513
# Full Hash: 66b279e5a513bf706129ac64cf44e3353e3a0d71
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 3eb8fdd0ba6d -r 66b279e5a513 dom/fetch/FetchStreamUtils.h
--- a/dom/fetch/FetchStreamUtils.h	Thu Jan 12 17:24:28 2023 +0000
+++ b/dom/fetch/FetchStreamUtils.h	Thu Jan 12 17:24:28 2023 +0000
@@ -40,7 +40,8 @@
 // process. Can only be called in the parent process.
 ParentToChildStream ToParentToChildStream(
     const NotNull<nsCOMPtr<nsIInputStream>>& aStream, int64_t aStreamSize,
-    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent);
+    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent,
+    bool aSerializeAsLazy = true);
 
 // Convert a ParentToParentStream to a ParentToChildStream. Can only be called
 // in the parent process.